src/client-entry.ts

```diff
@@ -4,6 +4,11 @@ import store from '@/store';
 
 Vue.config.productionTip = false;
 
+declare const mw: any;
+
+store.state.languageKey = mw.uls.getBrowserLanguage();
+store.state.termData = mw.config.get( 'wbEntity' );
```

Comment:
Only labels, descriptions and aliases are needed in the store, so not the whole blob should be added to the store, especially since we will likely only send the needed data (again, only labels, descriptions, aliases) with the server side request.

If we do a `JSON.parse( mw.config.get( 'wbEntity' ) )` then we can separately add `id`, `labels`, `descriptions`, and `aliases` to the store. This could even be extracted into a small class that gets the blob as an input and fills the store, so it's easier to test.